conftest.py

```diff
@@ -14,16 +14,10 @@
               "widgets", "exporters", "sortingcomponents", "generation"]
 
 
-# define global test folder
-def pytest_sessionstart(session):
-    # setup_stuff
-    pytest.global_test_folder = Path(__file__).parent / "test_folder"
-    if pytest.global_test_folder.is_dir():
-        shutil.rmtree(pytest.global_test_folder)
-    pytest.global_test_folder.mkdir()
-
-    for mark_name in mark_names:
-        (pytest.global_test_folder / mark_name).mkdir()
+@pytest.fixture
```

Comment:
@paulrignanese @JoeZiminski moving the fixture here allows us to remove it from everywhere else, since it's available to the entire pytest session ;)